lib/Controller/ConfigController.php

```diff
@@ -145,14 +146,14 @@ public function setConfig(array $values): DataResponse {
 	 *
 	 * @param array<string, string|null> $values
 	 *
-	 * @return DataResponse
-	 * @throws OpenprojectErrorException
+	 * @return array<string, bool|string>
+	 * @throws \Exception
 	 */
-	public function setAdminConfig(array $values): DataResponse {
+	public function setUpIntegrationConfig(array $values): array {
```

Comment:
only used in the own class
```suggestion
	private function setIntegrationConfig(array $values): array {
```